projects/habitat_ovmm/eval_baselines_agent.py

```diff
@@ -67,19 +69,24 @@
     )
 
     # get baseline config
-    baseline_config = get_ovmm_baseline_config(args.baseline_config_path)
+    baseline_config = get_omega_config(args.baseline_config_path)
+
+    # get env config
+    env_config = get_omega_config(args.env_config_path)
```

Comment:
Why is the habitat config coming through an os path? If it's because of something related to the challenge, should the env_config also come through the os path?

Reply:
Not sure about these. @rpartsey any idea?